xcat-inventory/xcclient/inventory/manager.py

```diff
@@ -609,4 +651,9 @@ def importobj(srcfile,srcdir,objtype,objnames=None,dryrun=None,version=None,upda
           
 
               
-                 
+def envlist():
+    print("#<%s> : %s#\n"%('Notice','refer the variables in the inventory file with format "{{variable name}}"'))
```

Comment:
Why use such strange format for the `NOTICE` line?  looks like a free style.

Reply:
this is to differentiate the `Notice` with the variable list